package.json

```diff
@@ -24,17 +24,17 @@
     "build:types": "tsc",
     "build:esm": "esbuild src/js/*.js src/js/*/*.js src/js/*/*/*.js --target=es2019 --format=esm --outdir=dist",
     "build:cjs": "esbuild src/js/*.js src/js/*/*.js src/js/*/*/*.js --target=es2019 --format=cjs --outdir=dist/cjs",
-    "build:iife": "esbuild src/js/index.js --bundle --target=es2019 --format=iife --outdir=dist/iife --minify --sourcemap --global-name=MediaChrome && yarn size",
+    "build:iife": "esbuild src/js/index.js --bundle --target=es2019 --format=iife --outdir=dist/iife --global-name=MediaChrome",
     "build:react": "node ./scripts/react/build.js",
     "build": "run-s build:types build:esm build:cjs build:iife build:react",
     "watch": "run-p watch:types watch:esbuild",
     "watch:types": "yarn build:types -w",
-    "watch:esbuild": "node ./scripts/esbuild.js",
-    "dev": "yarn watch",
+    "watch:esbuild": "yarn build:esm --watch=forever",
+    "serve": "npx serve . --config vercel.json --cors",
```

Comment:
can serve be added as a dev dependency? Then we won't even need to use `npx` here.